nrich-notification-spring-boot-starter/README.md

```diff
@@ -18,53 +22,50 @@ The artifact is published on [Maven Central Repository](https://search.maven.org
 With Maven:
 
 ```xml
-
 <dependency>
     <groupId>net.croz.nrich</groupId>
     <artifactId>nrich-notification-spring-boot-starter</artifactId>
     <version>${nrich.version}</version>
 </dependency>
-
 ```
 
 With Gradle:
 
 ```groovy
-
 implementation "net.croz.nrich:nrich-notification-spring-boot-starter:${nrich.version}"
-
 ```
 
 Note if using `nrich-bom` dependency versions should be omitted.
 
 ### Configuration
 
-Configuration is done through a property file, available properties and descriptions are given bellow (all properties are prefixed with nrich.notification which is omitted for readability):
+The configuration is done through a property file.
+Available properties and descriptions are given bellow (all properties are prefixed with **nrich.notification** which is omitted for readability):
 
 | property            | description                                                                                                      | default value |
 |---------------------|------------------------------------------------------------------------------------------------------------------|---------------|
-| register-messages   | Whether default messages should be registered (if users want to provide their own messages this can be disabled) | true          |
+| register-messages   | whether default messages should be registered (if users want to provide their own messages this can be disabled) | true          |
 
-The default configuration values in yaml format for easier modification are given bellow:
+The default configuration values are given bellow in a yaml format for easier modification:
 
 ```yaml
-
 nrich.notification:
-  register-messages: true
-
+    register-messages: true
 ```
 
-The standard notification messages are given in `notificationMessages` file and contain default error, success and info messages that will be returned to the client if no other messages are
-configured.
+The standard notification messages are given in the `notificationMessages` file which contains default error,
```

Comment:
Could you also update this, the name of the file has changed and is now nrich-notification-messages. And maybe a link to that file also?

Reply:
Done!